frontend/src/app/supportAdmin/PendingOrganizations/PendingOrganizations.tsx

```diff
@@ -28,17 +35,22 @@ const phoneUtil = PhoneNumberUtil.getInstance();
 const PendingOrganizations = ({
   organizations,
   submitIdentityVerified,
+  submitDeletion,
   loading,
   refetch,
   showNotification,
 }: Props) => {
   const [orgToVerify, setOrgToVerify] = useState<PendingOrganization | null>(
     null
   );
+
+  const [orgToDelete, setOrgToDelete] = useState<PendingOrganization | null>(
+    null
+  );
   const [verifyInProgress, setVerifyInProgress] = useState<boolean>(false);
   const [isUpdating, setIsUpdating] = useState(false);
   const [editOrg] = useEditPendingOrganizationMutation();
-  // should be removed once mutation for deletion of pending orgs is written
```

Comment:
nit: I would keep this comment in place, because this mutation won't be applicable to the old type of pending orgs - we're going to have to do a custom job to remove those.

Reply:
thanks for the flag! updated the comment with the relevant context.